src/portal_visualization/builders/base_builders.py

```diff
@@ -2,21 +2,40 @@
 from collections import namedtuple
 from abc import ABC, abstractmethod
 
+import nbformat
 
-ConfCells = namedtuple('ConfCells', ['conf', 'cells'])
+
+ConfigsCells = namedtuple('ConfigsCells', ['configs', 'cells'])
+
+
+def _get_cells_from_conf_list(confs):
+    cells = []
+    if len(confs) > 1:
+        cells.append(nbformat.v4.new_markdown_cell('Multiple visualizations are available.'))
+    for conf in confs:
+        cells.extend(_get_cells_from_conf(conf))
+    return cells
+
+
+def _get_cells_from_conf(conf):
+    imports, conf_expression = conf.to_python()
+    return [
+        nbformat.v4.new_code_cell(f'from vitessce import {", ".join(imports)}'),
+        nbformat.v4.new_code_cell(f'conf = {conf_expression}\nconf.widget()'),
+    ]
 
 
 class NullViewConfBuilder():
-    def __init__(self, entity, groups_token, assets_endpoint, **kwargs):
+    def __init__(self, entity, groups_token, assets_endpoint):
         # Just so it has the same signature as the other builders
         pass
 
-    def get_conf_cells(self, **kwargs):
-        return ConfCells(None, None)
+    def get_configs_cells(self, marker=None):
+        return ConfigsCells([], [])
```

Comment:
For consistency, even for the Null case we return lists. (One possibility for the cells would be to generate a message that there is no visualization... Sort of depends on the direction taken by workspaces.)